tests/unit/local/lambdafn/test_runtime.py

```diff
@@ -259,9 +256,6 @@ def test_must_run_container_and_wait_for_result(self, LambdaContainerMock):
 
         self.runtime.invoke(self.func_config, event, debug_context=debug_options, stdout=stdout, stderr=stderr)
 
-        # Verify if Lambda Event data is set
-        self.env_vars.add_lambda_event_body.assert_called_with(event)
```

Comment:
Same here, remove the `event` var created on line 233

Reply:
event is used in invoke function